stdlib/3/unittest.pyi

```diff
@@ -15,8 +15,8 @@ _FT = TypeVar('_FT', Callable[[Any], Any])
 
 
 def skip(reason: str) -> Callable[[_FT], _FT]: ...
-def skipIf(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
-def skipUnless(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
+def skipIf(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
+def skipUnless(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
```

Comment:
Because this is a stub, there's really no difference between leaving the type out, using Any, or using object. I like object because the truthiness check is actually defined there (other objects just override it). But I'm flexible. What's your preference?